openpype/pipeline/frames.py

```diff
@@ -0,0 +1,79 @@
+"""Utility functions.
+
+This module contains utility functions dedicated to frame range operations
+"""
+import clique
+
+from openpype.lib import Logger
+
+log = Logger.get_logger(__name__)
+
+
+def get_frame_start_str(frame_start, frame_end):
+    """Get frame start string.
+
+    Args:
+        frame_start (int): first frame
+        frame_end (int): last frame
+
+    Returns:
+        str: frame start string
+    """
+    padding = len(str(frame_end))
+    return str(frame_start).zfill(padding)
+
+
+def get_asset_frame_range(asset_name=None, asset_id=None, fields=None):
+    """Get the current assets frame range and handles.
+
+    Args:
+        asset_name Optional[str]: Name of asset used for filter.
+        asset_id Optional[Union[str, ObjectId]]: Asset document id.
+            If entered then is used as only filter.
+        fields Optional[Union[List[str], None]]: Limit returned data
+            of asset documents to specific keys.
+    Returns:
+        dict: with frame start, frame end, handle start, handle end.
+    """
+    from openpype.pipeline.context_tools import get_current_project_asset
+
+    # Set frame start/end
+    asset = get_current_project_asset()
```

Comment:
We only need a few fields, right?
```suggestion
    asset = get_current_project_asset(fields=["data.frameStart", "data.frameEnd", "data.handleStart", "data.handleEnd"])
```